x/authz/spec/README.md

```diff
@@ -0,0 +1,26 @@
+<!--
+order: 0
+title: Authz Overview
+parent:
+  title: "authz"
+-->
+
+# `authz`
+
+## Contents
+
+## Abstract
+`x/authz` is an implementation of a Cosmos SDK module, per [ADR 30](docs/architecture/adr-030-authz-module.md), that allows 
```

Comment:
```suggestion
`x/authz` is an implementation of a Cosmos SDK module, per [ADR 30](../../../architecture/adr-030-authz-module.md), that allows 
```

Reply:
Same as above, we need to change the relative path to have this render properly when viewing on docs.cosmos.network.